From 9b951f5a5df3cf9a0e88cce699374082318095c5 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Thu, 28 Aug 2014 14:22:31 -0400
Subject: [cpufreq] don't break string in print statements

Message-id: <1409235879-1283-58-git-send-email-prarit@redhat.com>
Patchwork-id: 89522
O-Subject: [RHEL7.1 PATCH BZ 1134639 057/185] cpufreq: don't break string in print statements
Bugzilla: 1134369
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Mark Langsdorf <mlangsdo@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1134639

commit 75949c9a1fe0fd07983788449059337edac2b9f6
Author: Viresh Kumar <viresh.kumar@linaro.org>
Date:   Wed Oct 2 14:13:13 2013 +0530

    cpufreq: don't break string in print statements

    As a rule its better not to break string (quoted inside "") in a
    print statement even if it crosses 80 column boundary as that may
    introduce bugs and so this patch rewrites one of the print statements..

    Signed-off-by: Viresh Kumar <viresh.kumar@linaro.org>
    Reviewed-by: Srivatsa S. Bhat <srivatsa.bhat@linux.vnet.ibm.com>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Cc: Lenny Szubowicz <lszubowi@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/cpufreq/cpufreq.c b/drivers/cpufreq/cpufreq.c
index b510049..aa83a3f 100644
--- a/drivers/cpufreq/cpufreq.c
+++ b/drivers/cpufreq/cpufreq.c
@@ -1205,8 +1205,8 @@ static int __cpufreq_remove_dev_prepare(struct device *dev,
    update_policy_cpu(policy, new_cpu);
 
    if (!frozen) {
-    pr_debug("%s: policy Kobject moved to cpu: %d "
-      "from: %d\n",__func__, new_cpu, cpu);
+    pr_debug("%s: policy Kobject moved to cpu: %d from: %d\n",
+      __func__, new_cpu, cpu);
    }
   }
  }
-- 
1.7.1